source/parser/comptime.civet

```diff
@@ -114,6 +120,13 @@ function serialize(value: ???): string
                 for own key, value in val as {[key: string]: ???}
                   `${JSON.stringify key}:${recurse value}`
               ).join(",") + "}"
+            when Int8Array::, Uint8Array::, Int16Array::, Uint16Array::, Int32Array::, Uint32Array::, Float32Array::, Float64Array::, Uint8ClampedArray::
+              // There's no "TypedArray" interface in TS
+              `new ${val.constructor.name}([${(val as any).join ','}])`
+            when BigInt64Array::, BigUint64Array::
+              `new ${val.constructor.name}([${Array.from(val as ArrayLike<bigint>, `${&}n`).join ','}])`
+            when Buffer::
```

Comment:
Can you test this on the browser build? (yarn docs:dev, click on Playground) I'm worried about what happens when this global isn't defined. Maybe `global.Buffer` works better?

Reply:
It seems to work fine, but I'm surprised that it does. I'll change this to `global.Buffer?::` or whatever the correct syntax is.

Reply:
So not only does that spelling not work (#1191), `global` doesn't exist in the browser, either!

Reply:
Hmm. `globalThis`?

Reply:
Yep, `globalThis` seemed to work. Thank you JavaScript very cool